src/dblib.ml

```diff
@@ -38,12 +38,20 @@ let err_handler (f: severity -> int -> string -> unit) =
 let default_err_handler severity _err msg =
   raise(Error(severity, msg))
 
+let msg_handler (f: string -> severity -> int -> unit) =
```

Comment:
Any reason why the order of the arguments is different from the `err_handler`?

Reply:
Fixed.